cli/healthcare_classifier_cli.py

```diff
@@ -0,0 +1,317 @@
+"""
+End-to-End Healthcare Classification CLI
+
+This provides a complete classification pipeline:
+1. First classifies as "medical" or "insurance"
+2. If medical, applies reason classification for detailed categorization
+
+IMPORTANT: Activate virtual environment first!
+Usage:
+  source .venv/bin/activate
+  python cli/healthcare_classifier_cli.py --interactive
+"""
+
+import argparse
+import json
+import sys
+from pathlib import Path
+
+# Add project root to path
+REPO_ROOT = Path(__file__).resolve().parents[1]
+if str(REPO_ROOT) not in sys.path:
+    sys.path.insert(0, str(REPO_ROOT))
+
+def classify_healthcare_query(query: str):
```

Comment:
Should we move this and the next two helper functions below into `classifier/utils.py` and then just import them in this file?